luigi_pipeline/seqr_loading.py

```diff
@@ -106,14 +102,24 @@ def read_vcf_write_mt(self, schema_cls=SeqrVariantsAndGenotypesSchema):
         # hgmd is optional.
         hgmd = hl.read_table(self.hgmd_ht_path) if self.hgmd_ht_path else None
 
-        mt = schema_cls(mt, ref_data=ref_data, clinvar_data=clinvar, hgmd_data=hgmd).annotate_all(
+        mt = schema_cls(mt, ref_data=ref_data, clinvar_data=clinvar, hgmd_data=hgmd, **kwargs).annotate_all(
```

Comment:
instead of passing through kwargs, which can get messy, use a helper method like `get_schema_class_kwargs` which here would be
```
def get_schema_class_kwargs(self):
        ref_data = hl.read_table(self.reference_ht_path)
        clinvar = hl.read_table(self.clinvar_ht_path)
        # hgmd is optional.
        hgmd = hl.read_table(self.hgmd_ht_path) if self.hgmd_ht_path else None
        return {'ref_data': ref_data, 'clinvar': clinvar, 'hgmd_data': hgmd}
```

and for the mito class it would be 
```
def get_schema_class_kwargs(self):
   kwargs = super().get_schema_class_kwargs()
   high_constraint_region = hl.import_locus_intervals(self.high_constraint_interval_path,
                                                           reference_genome='GRCh38')
   kwargs['high_constraint_region'] = high_constraint_region
```